README.md

```diff
@@ -48,3 +54,13 @@ yarn lint
 ### Customize configuration
 
 See [Configuration Reference](https://cli.vuejs.org/config/).
+
+### Deployment
+
+The app is deployed to [GitHub pages](https://epimorphics.github.com/rdf_explorer/).
+To update the deployment, ensure you don't have any uncommitted changes then run
+the `./deploy.sh` Bash script.
+
+**Note**: this script does do some force checkouts to update `gh-pages` branch, so
```

Comment:
Is it possible to be alerted to uncommitted changes and abort?

Reply:
> Is it possible to be alerted to uncommitted changes and abort?
Good suggestion. Will investigate